ui.apps/src/main/javascript/minicart/src/components/Minicart/container.js

```diff
@@ -11,23 +11,11 @@
  *    governing permissions and limitations under the License.
  *
  ******************************************************************************/
-import React, { useCallback } from 'react';
+import React from 'react';
 import MiniCart from './minicart';
 
-import { useGuestCart } from '../../utils/hooks';
-
 const Container = () => {
-    const [cartId, resetCart] = useGuestCart();
```

Comment:
The sole purpose of the `Container` component was to provide some data to the Minicart. If we're only going to render the `Minicart` component then it doesn't make sense to keep this component around.